src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -280,9 +280,15 @@ private static void printPropertyValue(String key, String value) {
     /*
      * prints the locale subopt/section
      */
-    private static void printLocale() {
+    private static void printLocale(boolean summaryMode) {
```

Comment:
Rename to printLocaleSummary (without an argument) and just print the summary.
(No help).
And drop the printing of the locales.